### PR TITLE
Final Review

### PR DESCRIPTION
**Pull request by awais-amjed**
This is a Giggy, A web Application with a small collection of Jokes.

Link to the video of Project [Demo](https://drive.google.com/file/d/17nS5Bx-tf-drFefibOtziAK-DOslSN2e/view?usp=sharing)

home_page_screenshot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,72 @@
...
+### Setup
+
+- Clone the GitHub Repository
+- Go to the Project Directory
+- Run ```npm install```
+- Run ```npm build``` to build the project
+
+### Usage
+
+- Run ```npm dev``` to start the live server
```
Comment (Meltrust) [inline README.md]: - [ ] Here, kindly correct the commands to `npm run build` and `npm run dev`, so the app is well documented 💪

File: src/unit_tests/comments_counter.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  json: () => Promise.resolve(mockData),
+}));
+
+describe('Get comments the the length', () => {
+  test('2 Comment added count should be 2', async () => {
+    await InvolvementAPI.getComments('test');
+
+    expect(InvolvementAPI.commentCount()).toBe(2);
+  });
+});
```
Comment (Meltrust) [inline src/unit_tests/comments_counter.test.js:25]: - [ ] This test is not passing:

image

Kindly correct this one.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @awais-amjed,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Beautiful, beautiful project✔️ 
- The app works ✔️
- Nice code organization✔️
- Nice use of webpack and modules ✔️ 
- You went overboard the requirements!! 👏  

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/unit_tests/comments_counter.test.js (hunk 2 above)
Comment (Kingstalux) [reply src/unit_tests/comments_counter.test.js:25]: resolved 👍

File: README.md (hunk 1 above)
Comment (Kingstalux) [reply README.md]: resolved 👍

**Review by Kingstalux — APPROVED** [review]
Hi @awais-amjed and @Bavon101 , 
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :ship: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**